<?xml version='1.0' encoding='utf-8'?>
<testsuites tests="147" failures="23" errors="0" skipped="45" time="0.0"><testsuite name="SoftwareManagement.Plugin.apama_plugin_install" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>13:21:27 INFO  ==============================================================
13:21:27 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_install
13:21:27 INFO  Dir:   plugin_apama/
13:21:27 INFO  Title: Validate apama plugin install
13:21:27 INFO  ==============================================================
13:21:27 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
13:21:27 WARN  Aborted test due to SKIPPED outcome
13:21:27 INFO  
13:21:27 INFO  Test duration: 0.05 secs
13:21:27 INFO  Test final outcome:  SKIPPED
13:21:27 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
13:21:27 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_mon_support" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_mon_support
13:21:28 INFO  Dir:   plugin_apama/
13:21:28 INFO  Title: Validate mon file support of apama plugin
13:21:28 INFO  ==============================================================
13:21:28 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.03 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_remove
13:21:28 INFO  Dir:   plugin_apama/
13:21:28 INFO  Title: Validate apama plugin remove
13:21:28 INFO  ==============================================================
13:21:28 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.02 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_update" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
13:21:28 INFO  Dir:   plugin_apama/
13:21:28 INFO  Title: Validate apama plugin updating one project with another
13:21:28 INFO  ==============================================================
13:21:28 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.02 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>14:12:14 INFO  ==============================================================
14:12:14 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
14:12:14 INFO  Dir:   plugin_apama/
14:12:14 INFO  Title: Validate apama plugin updating one project with another
14:12:14 INFO  ==============================================================
14:12:14 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
14:12:15 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
14:12:16 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
14:12:16 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
14:12:17 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
14:12:18 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
14:12:18 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
14:12:18 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
14:12:18 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
14:12:19 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
14:12:19 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

14:12:19 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
14:12:19 WARN  Aborted test due to BLOCKED outcome
14:12:19 INFO  
14:12:19 INFO  cleanup:
14:12:19 INFO  Removing apama project
14:12:35 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
14:12:36 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
14:12:36 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
14:12:36 INFO  
14:12:36 INFO  Test duration: 21.92 secs
14:12:36 INFO  Test final outcome:  BLOCKED
14:12:36 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
14:12:36 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>14:15:35 INFO  ==============================================================
14:15:35 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
14:15:35 INFO  Dir:   plugin_apama/
14:15:35 INFO  Title: Validate apama plugin updating one project with another
14:15:35 INFO  ==============================================================
14:15:35 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
14:15:35 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
14:15:36 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
14:15:37 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
14:15:37 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
14:15:38 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
14:15:38 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
14:15:39 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
14:15:39 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
14:15:40 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
14:15:40 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

14:15:40 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
14:15:40 WARN  Aborted test due to BLOCKED outcome
14:15:40 INFO  
14:15:40 INFO  cleanup:
14:15:40 INFO  Removing apama project
14:15:56 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
14:15:56 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
14:15:57 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
14:15:57 INFO  
14:15:57 INFO  Test duration: 21.95 secs
14:15:57 INFO  Test final outcome:  BLOCKED
14:15:57 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
14:15:57 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_download_path" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Install package to directory specified in tedge config set tmp.path
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.07 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>13:54:11 INFO  ==============================================================
13:54:11 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
13:54:11 INFO  Dir:   software_management_end_to_end/
13:54:11 INFO  Title: Install package to directory specified in tedge config set tmp.path
13:54:11 INFO  ==============================================================
13:54:11 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
13:54:11 INFO  EnvironmentC8y Setup
13:54:12 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
13:54:27 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:54:29 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
13:54:30 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
13:54:30 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
13:54:33 INFO  State of current operation: SUCCESSFUL
13:54:33 INFO  Assert that (False == value) with value=False ... passed
13:54:33 INFO  Executed sudo, exit status 0
13:54:34 INFO  Executed sudo, exit status 0
13:54:35 INFO  Executed sudo, exit status 0
13:54:39 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:54:39 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
13:54:39 INFO  Delaying execution by 8 seconds
13:55:01 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
13:55:01 INFO  Response status: 201
13:55:01 INFO  Response to action: {
    "creationTime": "2022-05-20T13:55:01.318Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19422041",
    "id": "19422041",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
13:55:01 INFO  Started operation: {'creationTime': '2022-05-20T13:55:01.318Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19422041', 'id': '19422041', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
13:55:01 WARN  , retrying in 0.5 seconds...
13:55:01 WARN  , retrying in 0.5 seconds...
13:55:02 WARN  , retrying in 0.5 seconds...
13:55:02 WARN  , retrying in 0.5 seconds...
13:55:03 WARN  , retrying in 0.5 seconds...
13:55:03 WARN  , retrying in 0.5 seconds...
13:55:04 WARN  , retrying in 0.5 seconds...
13:55:04 WARN  , retrying in 0.5 seconds...
13:55:05 WARN  , retrying in 0.5 seconds...
13:55:05 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
13:55:06 WARN  AssertionError: ... blocked 
13:55:06 INFO  
13:55:06 INFO  cleanup:
13:55:07 INFO  Executed sudo, exit status 0
13:55:07 INFO  Executed sudo, exit status 0
13:55:11 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:55:12 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
13:55:12 INFO  Delaying execution by 7 seconds
13:55:33 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:55:37 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:55:38 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
13:55:38 INFO  
13:55:38 INFO  Test duration: 86.37 secs
13:55:38 INFO  Test final outcome:  BLOCKED
13:55:38 INFO  Test outcome reason: AssertionError:
13:55:38 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>14:15:57 INFO  ==============================================================
14:15:57 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
14:15:57 INFO  Dir:   software_management_end_to_end/
14:15:57 INFO  Title: Install package to directory specified in tedge config set tmp.path
14:15:57 INFO  ==============================================================
14:15:58 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:15:58 INFO  EnvironmentC8y Setup
14:15:59 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:16:13 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:16:14 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:16:14 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:16:15 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:16:17 INFO  State of current operation: FAILED
14:16:17 INFO  Assert that (False == value) with value=False ... passed
14:16:18 INFO  Executed sudo, exit status 0
14:16:18 INFO  Executed sudo, exit status 0
14:16:19 INFO  Executed sudo, exit status 0
14:16:23 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:16:23 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
14:16:23 INFO  Delaying execution by 8 seconds
14:16:45 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
14:16:45 INFO  Response status: 201
14:16:45 INFO  Response to action: {
    "creationTime": "2022-05-20T14:16:45.338Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426201",
    "id": "19426201",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
14:16:45 INFO  Started operation: {'creationTime': '2022-05-20T14:16:45.338Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426201', 'id': '19426201', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
14:16:45 WARN  , retrying in 0.5 seconds...
14:16:45 WARN  , retrying in 0.5 seconds...
14:16:46 WARN  , retrying in 0.5 seconds...
14:16:46 WARN  , retrying in 0.5 seconds...
14:16:47 WARN  , retrying in 0.5 seconds...
14:16:47 WARN  , retrying in 0.5 seconds...
14:16:48 WARN  , retrying in 0.5 seconds...
14:16:48 WARN  , retrying in 0.5 seconds...
14:16:49 WARN  , retrying in 0.5 seconds...
14:16:49 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
14:16:49 WARN  AssertionError: ... blocked 
14:16:49 INFO  
14:16:49 INFO  cleanup:
14:16:50 INFO  Executed sudo, exit status 0
14:16:51 INFO  Executed sudo, exit status 0
14:16:55 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:16:55 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
14:16:55 INFO  Delaying execution by 8 seconds
14:17:17 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:17:22 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:17:22 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:17:22 INFO  
14:17:22 INFO  Test duration: 85.40 secs
14:17:22 INFO  Test final outcome:  BLOCKED
14:17:22 INFO  Test outcome reason: AssertionError:
14:17:22 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_fail" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_fail
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for a failing installation
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_remove
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the apt plugin for installation and removal of packages
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_default" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_remove_default
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the default plugin for installation and removal of packages
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True" type="FAILED" />
		<system-out>14:02:29 INFO  ==============================================================
14:02:29 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
14:02:29 INFO  Dir:   software_management_end_to_end/
14:02:29 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
14:02:29 INFO  ==============================================================
14:02:29 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:02:29 INFO  EnvironmentC8y Setup
14:02:29 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:02:29 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
14:02:29 INFO  Delaying execution by 7 seconds
14:02:51 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:02:51 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:02:52 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:02:52 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:02:54 INFO  State of current operation: SUCCESSFUL
14:02:55 INFO  Assert that (False == value) with value=False ... passed
14:02:55 INFO  Assert that (False == value) with value=False ... passed
14:02:55 INFO  Assert that (False == value) with value=False ... passed
14:02:55 INFO  Package robotfindskitten is installed
14:02:55 WARN  Assert that (False == value) with value=True ... failed 
14:02:55 INFO  Response status: 201
14:02:55 INFO  Response to action: {
    "creationTime": "2022-05-20T14:02:55.919Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426103",
    "id": "19426103",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:02:55 INFO  Started operation: {'creationTime': '2022-05-20T14:02:55.919Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426103', 'id': '19426103', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
14:02:58 INFO  State of operation 19426103 : PENDING
14:03:00 INFO  State of operation 19426103 : EXECUTING
14:03:02 INFO  State of operation 19426103 : EXECUTING
14:03:04 INFO  State of operation 19426103 : EXECUTING
14:03:07 INFO  State of operation 19426103 : EXECUTING
14:03:09 INFO  State of operation 19426103 : EXECUTING
14:03:11 INFO  State of operation 19426103 : EXECUTING
14:03:15 INFO  State of operation 19426103 : EXECUTING
14:03:17 INFO  State of operation 19426103 : EXECUTING
14:03:19 INFO  State of operation 19426103 : EXECUTING
14:03:22 INFO  State of operation 19426103 : EXECUTING
14:03:24 INFO  State of operation 19426103 : EXECUTING
14:03:26 INFO  State of operation 19426103 : EXECUTING
14:03:28 INFO  State of operation 19426103 : EXECUTING
14:03:31 INFO  State of operation 19426103 : EXECUTING
14:03:33 INFO  State of operation 19426103 : EXECUTING
14:03:35 INFO  State of operation 19426103 : EXECUTING
14:03:38 INFO  State of operation 19426103 : EXECUTING
14:03:40 INFO  State of operation 19426103 : EXECUTING
14:03:42 INFO  State of operation 19426103 : EXECUTING
14:03:45 INFO  State of operation 19426103 : EXECUTING
14:03:47 INFO  State of operation 19426103 : EXECUTING
14:03:50 INFO  State of operation 19426103 : EXECUTING
14:03:52 INFO  State of operation 19426103 : EXECUTING
14:03:54 INFO  State of operation 19426103 : EXECUTING
14:03:56 INFO  State of operation 19426103 : EXECUTING
14:03:59 INFO  State of operation 19426103 : EXECUTING
14:04:01 INFO  State of operation 19426103 : EXECUTING
14:04:03 INFO  State of operation 19426103 : EXECUTING
14:04:05 INFO  State of operation 19426103 : EXECUTING
14:04:08 INFO  State of operation 19426103 : EXECUTING
14:04:10 INFO  State of operation 19426103 : EXECUTING
14:04:12 INFO  State of operation 19426103 : EXECUTING
14:04:15 INFO  State of operation 19426103 : EXECUTING
14:04:17 INFO  State of operation 19426103 : SUCCESSFUL
14:04:17 INFO  Assert that (False == value) with value=False ... passed
14:04:17 INFO  Package asciijump is installed
14:04:17 INFO  Assert that (True == value) with value=True ... passed
14:04:17 INFO  Package squirrel3 is installed
14:04:17 INFO  Assert that (True == value) with value=True ... passed
14:04:18 INFO  Package robotfindskitten is installed
14:04:18 INFO  Assert that (True == value) with value=True ... passed
14:04:18 INFO  Response status: 201
14:04:18 INFO  Response to action: {
    "creationTime": "2022-05-20T14:04:18.299Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426115",
    "id": "19426115",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:04:18 INFO  Started operation: {'creationTime': '2022-05-20T14:04:18.299Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426115', 'id': '19426115', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
14:04:20 INFO  State of operation 19426115 : PENDING
14:04:22 INFO  State of operation 19426115 : EXECUTING
14:04:25 INFO  State of operation 19426115 : EXECUTING
14:04:27 INFO  State of operation 19426115 : EXECUTING
14:04:29 INFO  State of operation 19426115 : EXECUTING
14:04:31 INFO  State of operation 19426115 : EXECUTING
14:04:34 INFO  State of operation 19426115 : EXECUTING
14:04:37 INFO  State of operation 19426115 : EXECUTING
14:04:39 INFO  State of operation 19426115 : EXECUTING
14:04:42 INFO  State of operation 19426115 : EXECUTING
14:04:44 INFO  State of operation 19426115 : EXECUTING
14:04:46 INFO  State of operation 19426115 : EXECUTING
14:04:48 INFO  State of operation 19426115 : EXECUTING
14:04:50 INFO  State of operation 19426115 : EXECUTING
14:04:53 INFO  State of operation 19426115 : EXECUTING
14:04:55 INFO  State of operation 19426115 : EXECUTING
14:04:57 INFO  State of operation 19426115 : EXECUTING
14:05:00 INFO  State of operation 19426115 : EXECUTING
14:05:03 INFO  State of operation 19426115 : EXECUTING
14:05:05 INFO  State of operation 19426115 : EXECUTING
14:05:07 INFO  State of operation 19426115 : EXECUTING
14:05:09 INFO  State of operation 19426115 : EXECUTING
14:05:11 INFO  State of operation 19426115 : EXECUTING
14:05:14 INFO  State of operation 19426115 : EXECUTING
14:05:17 INFO  State of operation 19426115 : EXECUTING
14:05:19 INFO  State of operation 19426115 : EXECUTING
14:05:21 INFO  State of operation 19426115 : EXECUTING
14:05:24 INFO  State of operation 19426115 : SUCCESSFUL
14:05:24 INFO  Assert that (False == value) with value=False ... passed
14:05:24 INFO  Assert that (False == value) with value=False ... passed
14:05:24 INFO  Assert that (False == value) with value=False ... passed
14:05:24 INFO  Assert that (False == value) with value=False ... passed
14:05:24 INFO  
14:05:24 INFO  cleanup:
14:05:28 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:05:29 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:05:29 INFO  
14:05:29 INFO  Test duration: 180.47 secs
14:05:29 INFO  Test final outcome:  FAILED
14:05:29 INFO  Test outcome reason: Assert that (False == value) with value=True
14:05:29 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True (+2 other failures)" type="FAILED" />
		<system-out>14:23:46 INFO  ==============================================================
14:23:46 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
14:23:46 INFO  Dir:   software_management_end_to_end/
14:23:46 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
14:23:46 INFO  ==============================================================
14:23:46 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:23:46 INFO  EnvironmentC8y Setup
14:23:47 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:23:47 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
14:23:47 INFO  Delaying execution by 6 seconds
14:24:07 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
14:24:08 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:24:08 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:24:09 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:24:11 INFO  State of current operation: SUCCESSFUL
14:24:11 INFO  Assert that (False == value) with value=False ... passed
14:24:11 INFO  Package asciijump is installed
14:24:12 WARN  Assert that (False == value) with value=True ... failed 
14:24:12 INFO  Package squirrel3 is installed
14:24:12 WARN  Assert that (False == value) with value=True ... failed 
14:24:12 INFO  Package robotfindskitten is installed
14:24:12 WARN  Assert that (False == value) with value=True ... failed 
14:24:12 INFO  Response status: 201
14:24:12 INFO  Response to action: {
    "creationTime": "2022-05-20T14:24:12.756Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426291",
    "id": "19426291",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:24:12 INFO  Started operation: {'creationTime': '2022-05-20T14:24:12.756Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426291', 'id': '19426291', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
14:24:15 INFO  State of operation 19426291 : PENDING
14:24:17 INFO  State of operation 19426291 : EXECUTING
14:24:19 INFO  State of operation 19426291 : EXECUTING
14:24:21 INFO  State of operation 19426291 : EXECUTING
14:24:23 INFO  State of operation 19426291 : EXECUTING
14:24:26 INFO  State of operation 19426291 : EXECUTING
14:24:28 INFO  State of operation 19426291 : EXECUTING
14:24:31 INFO  State of operation 19426291 : EXECUTING
14:24:33 INFO  State of operation 19426291 : EXECUTING
14:24:36 INFO  State of operation 19426291 : EXECUTING
14:24:38 INFO  State of operation 19426291 : EXECUTING
14:24:40 INFO  State of operation 19426291 : EXECUTING
14:24:42 INFO  State of operation 19426291 : EXECUTING
14:24:44 INFO  State of operation 19426291 : EXECUTING
14:24:47 INFO  State of operation 19426291 : EXECUTING
14:24:49 INFO  State of operation 19426291 : SUCCESSFUL
14:24:49 INFO  Assert that (False == value) with value=False ... passed
14:24:49 INFO  Package asciijump is installed
14:24:49 INFO  Assert that (True == value) with value=True ... passed
14:24:50 INFO  Package squirrel3 is installed
14:24:50 INFO  Assert that (True == value) with value=True ... passed
14:24:50 INFO  Package robotfindskitten is installed
14:24:50 INFO  Assert that (True == value) with value=True ... passed
14:24:50 INFO  Response status: 201
14:24:50 INFO  Response to action: {
    "creationTime": "2022-05-20T14:24:50.411Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426307",
    "id": "19426307",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:24:50 INFO  Started operation: {'creationTime': '2022-05-20T14:24:50.411Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426307', 'id': '19426307', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
14:24:52 INFO  State of operation 19426307 : PENDING
14:24:54 INFO  State of operation 19426307 : EXECUTING
14:24:57 INFO  State of operation 19426307 : EXECUTING
14:24:59 INFO  State of operation 19426307 : EXECUTING
14:25:01 INFO  State of operation 19426307 : EXECUTING
14:25:03 INFO  State of operation 19426307 : EXECUTING
14:25:06 INFO  State of operation 19426307 : EXECUTING
14:25:09 INFO  State of operation 19426307 : EXECUTING
14:25:11 INFO  State of operation 19426307 : EXECUTING
14:25:14 INFO  State of operation 19426307 : EXECUTING
14:25:16 INFO  State of operation 19426307 : EXECUTING
14:25:18 INFO  State of operation 19426307 : EXECUTING
14:25:20 INFO  State of operation 19426307 : EXECUTING
14:25:23 INFO  State of operation 19426307 : EXECUTING
14:25:25 INFO  State of operation 19426307 : EXECUTING
14:25:27 INFO  State of operation 19426307 : EXECUTING
14:25:30 INFO  State of operation 19426307 : EXECUTING
14:25:32 INFO  State of operation 19426307 : EXECUTING
14:25:35 INFO  State of operation 19426307 : EXECUTING
14:25:37 INFO  State of operation 19426307 : EXECUTING
14:25:39 INFO  State of operation 19426307 : EXECUTING
14:25:41 INFO  State of operation 19426307 : EXECUTING
14:25:44 INFO  State of operation 19426307 : EXECUTING
14:25:46 INFO  State of operation 19426307 : EXECUTING
14:25:49 INFO  State of operation 19426307 : EXECUTING
14:25:51 INFO  State of operation 19426307 : EXECUTING
14:25:53 INFO  State of operation 19426307 : EXECUTING
14:25:56 INFO  State of operation 19426307 : SUCCESSFUL
14:25:56 INFO  Assert that (False == value) with value=False ... passed
14:25:56 INFO  Assert that (False == value) with value=False ... passed
14:25:56 INFO  Assert that (False == value) with value=False ... passed
14:25:57 INFO  Assert that (False == value) with value=False ... passed
14:25:57 INFO  
14:25:57 INFO  cleanup:
14:26:01 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:26:02 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:26:02 INFO  
14:26:02 INFO  Test duration: 135.34 secs
14:26:02 INFO  Test final outcome:  FAILED
14:26:02 INFO  Test outcome reason: Assert that (False == value) with value=True (+2 other failures)
14:26:02 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple_mixed" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False" type="BLOCKED" />
		<system-out>14:05:29 INFO  ==============================================================
14:05:29 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
14:05:29 INFO  Dir:   software_management_end_to_end/
14:05:29 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
14:05:29 INFO  ==============================================================
14:05:29 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:05:29 INFO  EnvironmentC8y Setup
14:05:30 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:05:30 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
14:05:30 INFO  Delaying execution by 6 seconds
14:05:50 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:05:51 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:05:51 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:05:51 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:05:54 INFO  State of current operation: SUCCESSFUL
14:05:54 INFO  Assert that (False == value) with value=False ... passed
14:05:54 INFO  Assert that (False == value) with value=False ... passed
14:05:54 INFO  Assert that (False == value) with value=False ... passed
14:05:55 INFO  Assert that (False == value) with value=False ... passed
14:05:55 INFO  Assert that (False == value) with value=False ... passed
14:05:56 INFO  Response status: 201
14:05:56 INFO  Response to action: {
    "delivery": {
        "log": [],
        "time": "2022-05-20T14:05:56.533Z",
        "status": "PENDING"
    },
    "creationTime": "2022-05-20T14:05:56.515Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426128",
    "id": "19426128",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:05:56 INFO  Started operation: {'delivery': {'log': [], 'time': '2022-05-20T14:05:56.533Z', 'status': 'PENDING'}, 'creationTime': '2022-05-20T14:05:56.515Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426128', 'id': '19426128', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
14:05:58 INFO  State of operation 19426128 : PENDING
14:06:01 INFO  State of operation 19426128 : EXECUTING
14:06:03 INFO  State of operation 19426128 : EXECUTING
14:06:05 INFO  State of operation 19426128 : EXECUTING
14:06:07 INFO  State of operation 19426128 : EXECUTING
14:06:09 INFO  State of operation 19426128 : EXECUTING
14:06:12 INFO  State of operation 19426128 : EXECUTING
14:06:15 INFO  State of operation 19426128 : EXECUTING
14:06:17 INFO  State of operation 19426128 : EXECUTING
14:06:20 INFO  State of operation 19426128 : EXECUTING
14:06:22 INFO  State of operation 19426128 : EXECUTING
14:06:24 INFO  State of operation 19426128 : EXECUTING
14:06:26 INFO  State of operation 19426128 : EXECUTING
14:06:29 INFO  State of operation 19426128 : EXECUTING
14:06:31 INFO  State of operation 19426128 : EXECUTING
14:06:33 INFO  State of operation 19426128 : EXECUTING
14:06:35 INFO  State of operation 19426128 : EXECUTING
14:06:38 INFO  State of operation 19426128 : EXECUTING
14:06:40 INFO  State of operation 19426128 : EXECUTING
14:06:42 INFO  State of operation 19426128 : EXECUTING
14:06:45 INFO  State of operation 19426128 : EXECUTING
14:06:47 INFO  State of operation 19426128 : EXECUTING
14:06:50 INFO  State of operation 19426128 : EXECUTING
14:06:53 INFO  State of operation 19426128 : EXECUTING
14:06:55 INFO  State of operation 19426128 : EXECUTING
14:06:57 INFO  State of operation 19426128 : EXECUTING
14:07:00 INFO  State of operation 19426128 : EXECUTING
14:07:02 INFO  State of operation 19426128 : EXECUTING
14:07:05 INFO  State of operation 19426128 : EXECUTING
14:07:07 INFO  State of operation 19426128 : EXECUTING
14:07:09 INFO  State of operation 19426128 : EXECUTING
14:07:11 INFO  State of operation 19426128 : EXECUTING
14:07:14 INFO  State of operation 19426128 : EXECUTING
14:07:16 INFO  State of operation 19426128 : EXECUTING
14:07:18 INFO  State of operation 19426128 : EXECUTING
14:07:20 INFO  State of operation 19426128 : EXECUTING
14:07:23 INFO  State of operation 19426128 : EXECUTING
14:07:25 INFO  State of operation 19426128 : EXECUTING
14:07:28 INFO  State of operation 19426128 : EXECUTING
14:07:30 INFO  State of operation 19426128 : EXECUTING
14:07:33 INFO  State of operation 19426128 : EXECUTING
14:07:35 INFO  State of operation 19426128 : EXECUTING
14:07:37 INFO  State of operation 19426128 : EXECUTING
14:07:39 INFO  State of operation 19426128 : EXECUTING
14:07:42 INFO  State of operation 19426128 : EXECUTING
14:07:44 INFO  State of operation 19426128 : EXECUTING
14:07:46 INFO  State of operation 19426128 : EXECUTING
14:07:48 INFO  State of operation 19426128 : EXECUTING
14:07:51 INFO  State of operation 19426128 : EXECUTING
14:07:54 INFO  State of operation 19426128 : EXECUTING
14:07:56 INFO  State of operation 19426128 : EXECUTING
14:07:58 INFO  State of operation 19426128 : EXECUTING
14:08:00 INFO  State of operation 19426128 : EXECUTING
14:08:03 INFO  State of operation 19426128 : EXECUTING
14:08:05 INFO  State of operation 19426128 : EXECUTING
14:08:07 INFO  State of operation 19426128 : EXECUTING
14:08:10 INFO  State of operation 19426128 : EXECUTING
14:08:12 INFO  State of operation 19426128 : EXECUTING
14:08:14 INFO  State of operation 19426128 : EXECUTING
14:08:17 INFO  State of operation 19426128 : EXECUTING
14:08:19 INFO  State of operation 19426128 : EXECUTING
14:08:21 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
14:08:22 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
14:08:22 INFO  
14:08:22 INFO  cleanup:
14:08:26 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:08:27 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:08:27 INFO  
14:08:27 INFO  Test duration: 177.93 secs
14:08:27 INFO  Test final outcome:  BLOCKED
14:08:27 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False
14:08:27 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)" type="BLOCKED" />
		<system-out>14:26:02 INFO  ==============================================================
14:26:02 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
14:26:02 INFO  Dir:   software_management_end_to_end/
14:26:02 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
14:26:02 INFO  ==============================================================
14:26:02 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:26:02 INFO  EnvironmentC8y Setup
14:26:02 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:26:02 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
14:26:02 INFO  Delaying execution by 6 seconds
14:26:22 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
14:26:23 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:26:24 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:26:24 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:26:26 INFO  State of current operation: SUCCESSFUL
14:26:26 INFO  Assert that (False == value) with value=False ... passed
14:26:27 INFO  Assert that (False == value) with value=False ... passed
14:26:27 INFO  Assert that (False == value) with value=False ... passed
14:26:27 INFO  Assert that (False == value) with value=False ... passed
14:26:27 INFO  Package moon-buggy is installed
14:26:27 WARN  Assert that (False == value) with value=True ... failed 
14:26:29 INFO  Response status: 201
14:26:29 INFO  Response to action: {
    "creationTime": "2022-05-20T14:26:29.092Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426320",
    "id": "19426320",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:26:29 INFO  Started operation: {'creationTime': '2022-05-20T14:26:29.092Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426320', 'id': '19426320', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
14:26:31 INFO  State of operation 19426320 : PENDING
14:26:33 INFO  State of operation 19426320 : EXECUTING
14:26:35 INFO  State of operation 19426320 : EXECUTING
14:26:38 INFO  State of operation 19426320 : EXECUTING
14:26:40 INFO  State of operation 19426320 : EXECUTING
14:26:42 INFO  State of operation 19426320 : EXECUTING
14:26:45 INFO  State of operation 19426320 : EXECUTING
14:26:47 INFO  State of operation 19426320 : EXECUTING
14:26:50 INFO  State of operation 19426320 : EXECUTING
14:26:52 INFO  State of operation 19426320 : EXECUTING
14:26:54 INFO  State of operation 19426320 : EXECUTING
14:26:57 INFO  State of operation 19426320 : EXECUTING
14:26:59 INFO  State of operation 19426320 : EXECUTING
14:27:01 INFO  State of operation 19426320 : EXECUTING
14:27:03 INFO  State of operation 19426320 : EXECUTING
14:27:06 INFO  State of operation 19426320 : EXECUTING
14:27:08 INFO  State of operation 19426320 : EXECUTING
14:27:10 INFO  State of operation 19426320 : EXECUTING
14:27:13 INFO  State of operation 19426320 : EXECUTING
14:27:15 INFO  State of operation 19426320 : EXECUTING
14:27:17 INFO  State of operation 19426320 : EXECUTING
14:27:20 INFO  State of operation 19426320 : EXECUTING
14:27:22 INFO  State of operation 19426320 : EXECUTING
14:27:24 INFO  State of operation 19426320 : EXECUTING
14:27:27 INFO  State of operation 19426320 : EXECUTING
14:27:29 INFO  State of operation 19426320 : EXECUTING
14:27:32 INFO  State of operation 19426320 : EXECUTING
14:27:34 INFO  State of operation 19426320 : EXECUTING
14:27:37 INFO  State of operation 19426320 : EXECUTING
14:27:39 INFO  State of operation 19426320 : EXECUTING
14:27:41 INFO  State of operation 19426320 : EXECUTING
14:27:43 INFO  State of operation 19426320 : EXECUTING
14:27:46 INFO  State of operation 19426320 : EXECUTING
14:27:48 INFO  State of operation 19426320 : EXECUTING
14:27:50 INFO  State of operation 19426320 : EXECUTING
14:27:52 INFO  State of operation 19426320 : EXECUTING
14:27:55 INFO  State of operation 19426320 : EXECUTING
14:27:57 INFO  State of operation 19426320 : EXECUTING
14:27:59 INFO  State of operation 19426320 : EXECUTING
14:28:02 INFO  State of operation 19426320 : EXECUTING
14:28:05 INFO  State of operation 19426320 : EXECUTING
14:28:07 INFO  State of operation 19426320 : EXECUTING
14:28:09 INFO  State of operation 19426320 : EXECUTING
14:28:12 INFO  State of operation 19426320 : EXECUTING
14:28:14 INFO  State of operation 19426320 : EXECUTING
14:28:16 INFO  State of operation 19426320 : EXECUTING
14:28:18 INFO  State of operation 19426320 : EXECUTING
14:28:21 INFO  State of operation 19426320 : EXECUTING
14:28:23 INFO  State of operation 19426320 : EXECUTING
14:28:26 INFO  State of operation 19426320 : EXECUTING
14:28:28 INFO  State of operation 19426320 : EXECUTING
14:28:30 INFO  State of operation 19426320 : EXECUTING
14:28:32 INFO  State of operation 19426320 : EXECUTING
14:28:35 INFO  State of operation 19426320 : EXECUTING
14:28:37 INFO  State of operation 19426320 : EXECUTING
14:28:39 INFO  State of operation 19426320 : EXECUTING
14:28:42 INFO  State of operation 19426320 : EXECUTING
14:28:45 INFO  State of operation 19426320 : EXECUTING
14:28:47 INFO  State of operation 19426320 : EXECUTING
14:28:49 INFO  State of operation 19426320 : EXECUTING
14:28:51 INFO  State of operation 19426320 : EXECUTING
14:28:53 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
14:28:54 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
14:28:54 INFO  
14:28:54 INFO  cleanup:
14:28:59 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:28:59 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:28:59 INFO  
14:28:59 INFO  Test duration: 177.45 secs
14:28:59 INFO  Test final outcome:  BLOCKED
14:28:59 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)
14:28:59 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_with_version" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:10 INFO  ==============================================================
13:31:10 INFO  Id:    SoftwareManagement.sm_apt_install_remove_with_version
13:31:10 INFO  Dir:   software_management_end_to_end/
13:31:10 INFO  Title: Validate end to end behaviour for the apt plugin for packages with version
13:31:10 INFO  ==============================================================
13:31:10 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:10 WARN  Aborted test due to SKIPPED outcome
13:31:10 INFO  
13:31:10 INFO  Test duration: 0.02 secs
13:31:10 INFO  Test final outcome:  SKIPPED
13:31:10 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:10 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
</testsuite><testsuite name="SoftwareManagement.sm_docker_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:10 INFO  ==============================================================
13:31:10 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
13:31:10 INFO  Dir:   software_management_end_to_end/
13:31:10 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
13:31:10 INFO  ==============================================================
13:31:10 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:10 WARN  Aborted test due to SKIPPED outcome
13:31:10 INFO  
13:31:10 INFO  Test duration: 0.02 secs
13:31:10 INFO  Test final outcome:  SKIPPED
13:31:10 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:10 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>14:13:11 INFO  ==============================================================
14:13:11 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
14:13:11 INFO  Dir:   software_management_end_to_end/
14:13:11 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
14:13:11 INFO  ==============================================================
14:13:12 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:13:12 INFO  EnvironmentC8y Setup
14:13:13 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:13:27 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:13:28 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:13:28 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:13:29 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:13:31 INFO  State of current operation: SUCCESSFUL
14:13:31 INFO  Response status: 201
14:13:31 INFO  Response to action: {
    "creationTime": "2022-05-20T14:13:31.507Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426191",
    "id": "19426191",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
14:13:31 INFO  Started operation: {'creationTime': '2022-05-20T14:13:31.507Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426191', 'id': '19426191', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
14:13:33 INFO  State of operation 19426191 : PENDING
14:13:35 INFO  State of operation 19426191 : EXECUTING
14:13:38 INFO  State of operation 19426191 : EXECUTING
14:13:40 INFO  State of operation 19426191 : EXECUTING
14:13:42 INFO  State of operation 19426191 : FAILED
14:13:42 ERROR Stopping as the operation has failed
14:13:42 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
14:13:42 WARN  SystemError: The operation has failed ... blocked 
14:13:42 INFO  
14:13:42 INFO  cleanup:
14:13:46 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:13:47 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:13:47 INFO  
14:13:47 INFO  Test duration: 35.77 secs
14:13:47 INFO  Test final outcome:  BLOCKED
14:13:47 INFO  Test outcome reason: SystemError: The operation has failed
14:13:47 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>14:31:36 INFO  ==============================================================
14:31:36 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
14:31:36 INFO  Dir:   software_management_end_to_end/
14:31:36 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
14:31:36 INFO  ==============================================================
14:31:36 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:31:36 INFO  EnvironmentC8y Setup
14:31:37 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:31:37 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
14:31:37 INFO  Delaying execution by 6 seconds
14:31:57 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:31:58 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:31:58 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:31:59 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:32:01 INFO  State of current operation: SUCCESSFUL
14:32:01 INFO  Response status: 201
14:32:01 INFO  Response to action: {
    "creationTime": "2022-05-20T14:32:01.717Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426375",
    "id": "19426375",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
14:32:01 INFO  Started operation: {'creationTime': '2022-05-20T14:32:01.717Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426375', 'id': '19426375', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
14:32:03 INFO  State of operation 19426375 : PENDING
14:32:06 INFO  State of operation 19426375 : EXECUTING
14:32:08 INFO  State of operation 19426375 : EXECUTING
14:32:10 INFO  State of operation 19426375 : EXECUTING
14:32:12 INFO  State of operation 19426375 : FAILED
14:32:12 ERROR Stopping as the operation has failed
14:32:12 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
14:32:12 WARN  SystemError: The operation has failed ... blocked 
14:32:12 INFO  
14:32:12 INFO  cleanup:
14:32:16 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:32:17 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:32:17 INFO  
14:32:17 INFO  Test duration: 40.54 secs
14:32:17 INFO  Test final outcome:  BLOCKED
14:32:17 INFO  Test outcome reason: SystemError: The operation has failed
14:32:17 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>13:31:10 INFO  ==============================================================
13:31:10 INFO  Id:    SoftwareManagement.sm_fake_install_remove
13:31:10 INFO  Dir:   software_management_end_to_end/
13:31:10 INFO  Title: Validate end to end behaviour for the dummy plugin for installation and removal of packages
13:31:10 INFO  ==============================================================
13:31:10 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
13:31:10 WARN  Aborted test due to SKIPPED outcome
13:31:10 INFO  
13:31:10 INFO  Test duration: 0.02 secs
13:31:10 INFO  Test final outcome:  SKIPPED
13:31:10 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
13:31:10 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove_multiple" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>13:31:10 INFO  ==============================================================
13:31:10 INFO  Id:    SoftwareManagement.sm_fake_install_remove_multiple
13:31:10 INFO  Dir:   software_management_end_to_end/
13:31:10 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages
13:31:10 INFO  ==============================================================
13:31:10 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
13:31:10 WARN  Aborted test due to SKIPPED outcome
13:31:10 INFO  
13:31:10 INFO  Test duration: 0.03 secs
13:31:10 INFO  Test final outcome:  SKIPPED
13:31:10 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
13:31:10 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple" />
</testsuite><testsuite name="SoftwareManagement.sm_mapper_fail_and_reconnect" tests="2" errors="0" failures="0" skipped="2" time="0"><testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>13:31:10 INFO  ==============================================================
13:31:10 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
13:31:10 INFO  Dir:   software_management_end_to_end/
13:31:10 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
13:31:10 INFO  ==============================================================
13:31:10 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
13:31:10 INFO  
13:31:10 INFO  Test duration: 0.02 secs
13:31:10 INFO  Test final outcome:  SKIPPED
13:31:10 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
13:31:10 INFO  
</system-out>
	</testcase>
<testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>14:33:51 INFO  ==============================================================
14:33:51 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
14:33:51 INFO  Dir:   software_management_end_to_end/
14:33:51 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
14:33:51 INFO  ==============================================================
14:33:51 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
14:33:51 INFO  
14:33:51 INFO  Test duration: 0.02 secs
14:33:51 INFO  Test final outcome:  SKIPPED
14:33:51 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
14:33:51 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_mixed_install_remove_multiple" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>13:31:10 INFO  ==============================================================
13:31:10 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
13:31:10 INFO  Dir:   software_management_end_to_end/
13:31:10 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
13:31:10 INFO  ==============================================================
13:31:10 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
13:31:10 WARN  Aborted test due to SKIPPED outcome
13:31:10 INFO  
13:31:10 INFO  Test duration: 0.02 secs
13:31:10 INFO  Test final outcome:  SKIPPED
13:31:10 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
13:31:10 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed (+1 other failures)" type="BLOCKED" />
		<system-out>14:33:51 INFO  ==============================================================
14:33:51 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
14:33:51 INFO  Dir:   software_management_end_to_end/
14:33:51 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
14:33:51 INFO  ==============================================================
14:33:51 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:33:51 INFO  EnvironmentC8y Setup
14:33:52 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:33:52 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
14:33:52 INFO  Delaying execution by 7 seconds
14:34:13 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
14:34:14 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:34:14 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:34:15 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:34:17 INFO  State of current operation: SUCCESSFUL
14:34:17 INFO  Package apple is installed
14:34:17 INFO  Assert that (True == value) with value=True ... passed
14:34:17 INFO  Package banana is installed
14:34:17 INFO  Assert that (True == value) with value=True ... passed
14:34:18 INFO  Package cherry is installed
14:34:18 INFO  Assert that (True == value) with value=True ... passed
14:34:18 INFO  Package asciijump is installed
14:34:18 WARN  Assert that (False == value) with value=True ... failed 
14:34:18 INFO  Assert that (False == value) with value=False ... passed
14:34:18 INFO  Response status: 201
14:34:18 INFO  Response to action: {
    "creationTime": "2022-05-20T14:34:18.850Z",
    "deviceId": "19408349",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426444",
    "id": "19426444",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "apple",
            "action": "install",
            "id": "5495053",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "banana",
            "action": "install",
            "id": "5494888",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "cherry",
            "action": "install",
            "id": "5495382",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "::docker",
            "url": " "
        }
    ]
}
14:34:18 INFO  Started operation: {'creationTime': '2022-05-20T14:34:18.850Z', 'deviceId': '19408349', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426444', 'id': '19426444', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'apple', 'action': 'install', 'id': '5495053', 'version': '::fruits', 'url': ' '}, {'name': 'banana', 'action': 'install', 'id': '5494888', 'version': '::fruits', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'cherry', 'action': 'install', 'id': '5495382', 'version': '::fruits', 'url': ' '}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '::docker', 'url': ' '}]}
14:34:21 INFO  State of operation 19426444 : PENDING
14:34:23 INFO  State of operation 19426444 : EXECUTING
14:34:25 INFO  State of operation 19426444 : EXECUTING
14:34:27 INFO  State of operation 19426444 : EXECUTING
14:34:30 INFO  State of operation 19426444 : EXECUTING
14:34:32 INFO  State of operation 19426444 : EXECUTING
14:34:34 INFO  State of operation 19426444 : EXECUTING
14:34:37 INFO  State of operation 19426444 : EXECUTING
14:34:39 INFO  State of operation 19426444 : EXECUTING
14:34:42 INFO  State of operation 19426444 : EXECUTING
14:34:44 INFO  State of operation 19426444 : EXECUTING
14:34:46 INFO  State of operation 19426444 : EXECUTING
14:34:48 INFO  State of operation 19426444 : EXECUTING
14:34:51 INFO  State of operation 19426444 : EXECUTING
14:34:53 INFO  State of operation 19426444 : EXECUTING
14:34:55 INFO  State of operation 19426444 : EXECUTING
14:34:57 INFO  State of operation 19426444 : FAILED
14:34:57 ERROR Stopping as the operation has failed
14:34:57 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_mixed_install_remove_multiple/run.py", line 95, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
14:34:57 WARN  SystemError: The operation has failed ... blocked 
14:34:57 INFO  
14:34:57 INFO  cleanup:
14:35:01 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:35:02 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:35:02 INFO  
14:35:02 INFO  Test duration: 70.42 secs
14:35:02 INFO  Test final outcome:  BLOCKED
14:35:02 INFO  Test outcome reason: SystemError: The operation has failed (+1 other failures)
14:35:02 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="agent_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="AgentInitSession" name="agent_init_session" />
</testsuite><testsuite name="apt_fail_noarg" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFailNoArg" name="apt_fail_noarg">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    apt_fail_noarg
13:21:28 INFO  Dir:   plugin_apt/
13:21:28 INFO  Title: Validate apt plugin fetures(prepare, list, install, remove finalize) without arguments
13:21:28 INFO  ==============================================================
13:21:28 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.05 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFailNoArg" name="apt_fail_noarg" />
</testsuite><testsuite name="apt_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFinalize" name="apt_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    apt_finalize
13:21:28 INFO  Dir:   plugin_apt/
13:21:28 INFO  Title: Validate apt plugin finalize - dense version
13:21:28 INFO  ==============================================================
13:21:28 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.03 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFinalize" name="apt_finalize" />
</testsuite><testsuite name="apt_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallDense" name="apt_install">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    apt_install
13:21:28 INFO  Dir:   plugin_apt/
13:21:28 INFO  Title: Validate apt plugin prepare - dense version
13:21:28 INFO  ==============================================================
13:21:28 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.03 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallDense" name="apt_install" />
</testsuite><testsuite name="apt_install_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFail" name="apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    apt_install_fail
13:21:28 INFO  Dir:   plugin_apt/
13:21:28 INFO  Title: Validate apt plugin prepare - failing
13:21:28 INFO  ==============================================================
13:21:28 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.02 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFail" name="apt_install_fail" />
</testsuite><testsuite name="apt_install_local_file" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    apt_install_local_file
13:21:28 INFO  Dir:   plugin_apt/
13:21:28 INFO  Title: Validate apt plugin install from local file - SUCCESS case
13:21:28 INFO  ==============================================================
13:21:28 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.02 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file" />
</testsuite><testsuite name="apt_install_local_file_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    apt_install_local_file_fail
13:21:28 INFO  Dir:   plugin_apt/
13:21:28 INFO  Title: Validate apt plugin install from local file - FAIL case
13:21:28 INFO  ==============================================================
13:21:28 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.02 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail" />
</testsuite><testsuite name="apt_install_local_file_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    apt_install_local_file_with_version
13:21:28 INFO  Dir:   plugin_apt/
13:21:28 INFO  Title: Validate apt plugin install from local file AND with a version - SUCCESS case
13:21:28 INFO  ==============================================================
13:21:28 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.02 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version" />
</testsuite><testsuite name="apt_install_local_file_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:28 INFO  ==============================================================
13:21:28 INFO  Id:    apt_install_local_file_with_version_fail
13:21:28 INFO  Dir:   plugin_apt/
13:21:28 INFO  Title: Validate apt plugin install from local file AND with a version - FAIL case
13:21:28 INFO  ==============================================================
13:21:28 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:28 WARN  Aborted test due to SKIPPED outcome
13:21:28 INFO  
13:21:28 INFO  Test duration: 0.02 secs
13:21:28 INFO  Test final outcome:  SKIPPED
13:21:28 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:28 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail" />
</testsuite><testsuite name="apt_list_format" tests="2" errors="0" failures="0" skipped="0" time="0"><testcase classname="AptPluginListFormat" name="apt_list_format" />
<testcase classname="AptPluginListFormat" name="apt_list_format" />
</testsuite><testsuite name="apt_prepare" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepare" name="apt_prepare">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_prepare
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin install - dense version
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepare" name="apt_prepare" />
</testsuite><testsuite name="apt_prepare_install_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_prepare_install_finalize
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin install use case
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize" />
</testsuite><testsuite name="apt_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveDense" name="apt_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_remove
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin prepare - dense version
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveDense" name="apt_remove" />
</testsuite><testsuite name="apt_remove_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveFails" name="apt_remove_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_remove_fail
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin prepare - failing
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveFails" name="apt_remove_fail" />
</testsuite><testsuite name="apt_sequence_install_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_sequence_install_with_version
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin install with version use case
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version" />
</testsuite><testsuite name="apt_sequence_install_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_sequence_install_with_version_fail
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin install with version fails use case
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail" />
</testsuite><testsuite name="apt_sequence_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_sequence_remove
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin prepare install remove finalize use case
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove" />
</testsuite><testsuite name="apt_sequence_remove_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_sequence_remove_with_version
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin remove with version use case
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version" />
</testsuite><testsuite name="apt_sequence_remove_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_sequence_remove_with_version_fail
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin remove with version fails when the version not found
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail" />
</testsuite><testsuite name="apt_update_list" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginUpdateList" name="apt_update_list">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_update_list
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin update-list command use case
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginUpdateList" name="apt_update_list" />
</testsuite><testsuite name="apt_wrong_option" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginWrongOption" name="apt_wrong_option">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    apt_wrong_option
13:21:32 INFO  Dir:   plugin_apt/
13:21:32 INFO  Title: Validate apt plugin install
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginWrongOption" name="apt_wrong_option" />
</testsuite><testsuite name="c8y_jwt_token_retrieval" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateJWTTokenRetrieval" name="c8y_jwt_token_retrieval" />
</testsuite><testsuite name="c8y_restart_bridge" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RestartBridge" name="c8y_restart_bridge" />
</testsuite><testsuite name="call_tedge_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-V" />
</testsuite><testsuite name="call_tedge_-h" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h" />
</testsuite><testsuite name="call_tedge_-h_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h_-V" />
</testsuite><testsuite name="call_tedge_config_help" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_help" />
</testsuite><testsuite name="call_tedge_config_list" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_list" />
</testsuite><testsuite name="custom_init_system" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="CustomInitSystem" name="custom_init_system" />
</testsuite><testsuite name="docker_plugin_finalize" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    docker_plugin_finalize
13:21:32 INFO  Dir:   plugin_docker/
13:21:32 INFO  Title: Validate docker plugin finalize command
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.05 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<failure message="BLOCKED: sudo&lt;docker_pull&gt; returned non-zero exit code 1" type="BLOCKED" />
		<system-out>14:12:40 INFO  ==============================================================
14:12:40 INFO  Id:    docker_plugin_finalize
14:12:40 INFO  Dir:   plugin_docker/
14:12:40 INFO  Title: Validate docker plugin finalize command
14:12:40 INFO  ==============================================================
14:12:41 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:12:41 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out ... passed
14:12:43 WARN  Executed sudo&lt;docker_pull&gt;, exit status 1
14:12:43 INFO  Contents of docker_pull.err: 
  Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  -----

14:12:44 WARN  sudo&lt;docker_pull&gt; returned non-zero exit code 1 ... blocked 
14:12:44 WARN  Aborted test due to BLOCKED outcome
14:12:44 INFO  
14:12:44 INFO  cleanup:
14:12:44 INFO  
14:12:44 INFO  Test duration: 3.94 secs
14:12:44 INFO  Test final outcome:  BLOCKED
14:12:44 INFO  Test outcome reason: sudo&lt;docker_pull&gt; returned non-zero exit code 1
14:12:44 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_install" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:32 INFO  ==============================================================
13:21:32 INFO  Id:    docker_plugin_install
13:21:32 INFO  Dir:   plugin_docker/
13:21:32 INFO  Title: Validate docker plugin install
13:21:32 INFO  ==============================================================
13:21:32 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:32 WARN  Aborted test due to SKIPPED outcome
13:21:32 INFO  
13:21:32 INFO  Test duration: 0.02 secs
13:21:32 INFO  Test final outcome:  SKIPPED
13:21:32 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:32 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>14:12:44 INFO  ==============================================================
14:12:44 INFO  Id:    docker_plugin_install
14:12:44 INFO  Dir:   plugin_docker/
14:12:44 INFO  Title: Validate docker plugin install
14:12:44 INFO  ==============================================================
14:12:44 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:12:44 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install/Output/linux/docker_images.out ... passed
14:12:47 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
14:12:47 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:23]
14:12:48 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
14:12:48 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
14:12:48 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install/Output/linux/docker_ps.out contains "registry" ... failed [environment_docker_plugin.py:49,run.py:26]
14:12:48 INFO  
14:12:48 INFO  cleanup:
14:12:48 INFO  Removing containers scheduled to be cleaned up: ['']
14:12:49 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
14:12:49 INFO  Removing images scheduled to be cleaned up: ['registry']
14:12:50 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
14:12:50 INFO  
14:12:50 INFO  Test duration: 6.49 secs
14:12:50 INFO  Test final outcome:  BLOCKED
14:12:50 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
14:12:50 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_install_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    docker_plugin_install_with_version
13:21:33 INFO  Dir:   plugin_docker/
13:21:33 INFO  Title: Validate docker plugin install
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>14:12:50 INFO  ==============================================================
14:12:50 INFO  Id:    docker_plugin_install_with_version
14:12:50 INFO  Dir:   plugin_docker/
14:12:50 INFO  Title: Validate docker plugin install
14:12:50 INFO  ==============================================================
14:12:51 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:12:51 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_images.out ... passed
14:12:53 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
14:12:53 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:24]
14:12:54 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
14:12:55 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
14:12:55 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps.out contains "registry:2.7.1" ... failed [environment_docker_plugin.py:49,run.py:28]
14:12:55 INFO  
14:12:55 INFO  cleanup:
14:12:55 INFO  Removing containers scheduled to be cleaned up: ['', '']
14:12:56 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
14:12:56 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
14:12:57 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
14:12:57 INFO  
14:12:57 INFO  Test duration: 6.55 secs
14:12:57 INFO  Test final outcome:  BLOCKED
14:12:57 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
14:12:57 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    docker_plugin_remove
13:21:33 INFO  Dir:   plugin_docker/
13:21:33 INFO  Title: Validate docker plugin remove
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>14:12:57 INFO  ==============================================================
14:12:57 INFO  Id:    docker_plugin_remove
14:12:57 INFO  Dir:   plugin_docker/
14:12:57 INFO  Title: Validate docker plugin remove
14:12:57 INFO  ==============================================================
14:12:58 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:12:58 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove/Output/linux/docker_images.out ... passed
14:13:00 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
14:13:00 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:latest' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

14:13:00 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
14:13:00 WARN  Aborted test due to BLOCKED outcome
14:13:00 INFO  
14:13:00 INFO  cleanup:
14:13:00 INFO  Removing containers scheduled to be cleaned up: ['', '']
14:13:01 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
14:13:01 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
14:13:01 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
14:13:01 INFO  
14:13:01 INFO  Test duration: 4.58 secs
14:13:01 INFO  Test final outcome:  BLOCKED
14:13:01 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
14:13:01 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    docker_plugin_remove_with_version
13:21:33 INFO  Dir:   plugin_docker/
13:21:33 INFO  Title: Validate docker plugin remove with version
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>14:13:01 INFO  ==============================================================
14:13:01 INFO  Id:    docker_plugin_remove_with_version
14:13:01 INFO  Dir:   plugin_docker/
14:13:01 INFO  Title: Validate docker plugin remove with version
14:13:01 INFO  ==============================================================
14:13:02 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:13:02 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove_with_version/Output/linux/docker_images.out ... passed
14:13:04 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
14:13:04 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.7.1' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

14:13:04 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
14:13:04 WARN  Aborted test due to BLOCKED outcome
14:13:04 INFO  
14:13:04 INFO  cleanup:
14:13:04 INFO  Removing containers scheduled to be cleaned up: ['', '']
14:13:05 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
14:13:05 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
14:13:06 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
14:13:06 INFO  
14:13:06 INFO  Test duration: 4.60 secs
14:13:06 INFO  Test final outcome:  BLOCKED
14:13:06 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
14:13:06 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_update" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    docker_plugin_update
13:21:33 INFO  Dir:   plugin_docker/
13:21:33 INFO  Title: Validate docker plugin install updates existing containers
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>14:13:06 INFO  ==============================================================
14:13:06 INFO  Id:    docker_plugin_update
14:13:06 INFO  Dir:   plugin_docker/
14:13:06 INFO  Title: Validate docker plugin install updates existing containers
14:13:06 INFO  ==============================================================
14:13:07 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:13:07 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_images.out ... passed
14:13:09 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
14:13:09 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.6.2' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

14:13:09 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
14:13:09 WARN  Aborted test due to BLOCKED outcome
14:13:09 INFO  
14:13:09 INFO  cleanup:
14:13:09 INFO  Removing containers scheduled to be cleaned up: ['', '']
14:13:10 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
14:13:10 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
14:13:11 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
14:13:11 INFO  
14:13:11 INFO  Test duration: 4.78 secs
14:13:11 INFO  Test final outcome:  BLOCKED
14:13:11 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
14:13:11 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="invalid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="InvalidDeviceId" name="invalid_device_id" />
</testsuite><testsuite name="local_mqtt_pub_fail_noqos" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_noqos" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos0" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos0" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos1" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos1" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos2" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos2" />
</testsuite><testsuite name="local_mqtt_pub_sub" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub" />
</testsuite><testsuite name="local_mqtt_pub_sub_retain" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPublishWithRetain" name="local_mqtt_pub_sub_retain" />
</testsuite><testsuite name="local_mqtt_pub_sub_with_sudo" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub_with_sudo" />
</testsuite><testsuite name="local_mqtt_sub_fail" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_sub_fail" />
</testsuite><testsuite name="log_request_error_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifySearchTextError" name="log_request_error_messages">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>13:18:27 INFO  ==============================================================
13:18:27 INFO  Id:    log_request_error_messages
13:18:27 INFO  Dir:   log_request_end_to_end/
13:18:27 INFO  Title: Validate end to end behaviour for the log request operation check for error messages
13:18:27 INFO  ==============================================================
13:18:28 INFO  EnvironmentC8y Setup
13:18:29 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
13:18:29 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 2 seconds
13:18:29 INFO  Delaying execution by 9 seconds
13:18:52 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:18:53 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
13:18:53 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
13:18:54 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
13:18:55 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
13:18:55 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
13:18:56 INFO  log response None
13:18:56 WARN  retry, retrying in 1 seconds...
13:18:57 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/19408351/binaries
13:18:58 WARN  Assert that (True == value) with value=False ... failed [run.py:43]
13:18:58 INFO  
13:18:58 INFO  cleanup:
13:18:58 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
13:18:59 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
13:18:59 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
13:19:03 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:19:04 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
13:19:04 INFO  
13:19:04 INFO  Test duration: 36.70 secs
13:19:04 INFO  Test final outcome:  FAILED
13:19:04 INFO  Test outcome reason: Assert that (True == value) with value=False
13:19:04 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="log_request_num_lines" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifyNumberOfLines" name="log_request_num_lines">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>13:19:04 INFO  ==============================================================
13:19:04 INFO  Id:    log_request_num_lines
13:19:04 INFO  Dir:   log_request_end_to_end/
13:19:04 INFO  Title: Validate end to end behaviour for the log request operation check for number of lines
13:19:04 INFO  ==============================================================
13:19:04 INFO  EnvironmentC8y Setup
13:19:05 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
13:19:05 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
13:19:05 INFO  Delaying execution by 6 seconds
13:19:26 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:19:26 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
13:19:27 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
13:19:27 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
13:19:28 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
13:19:28 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
13:19:29 INFO  log response None
13:19:29 WARN  retry, retrying in 1 seconds...
13:19:30 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/19408354/binaries
13:19:30 INFO  nlines 0
13:19:30 WARN  Assert that (True == value) with value=False ... failed [run.py:44]
13:19:30 INFO  
13:19:30 INFO  cleanup:
13:19:31 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
13:19:31 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
13:19:32 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
13:19:36 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:19:37 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
13:19:37 INFO  
13:19:37 INFO  Test duration: 32.87 secs
13:19:37 INFO  Test final outcome:  FAILED
13:19:37 INFO  Test outcome reason: Assert that (True == value) with value=False
13:19:37 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mapper_awaits_before_reconnect" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MapperReconnectAwait" name="mapper_awaits_before_reconnect">
		<failure message="FAILED: Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0" type="FAILED" />
		<system-out>13:34:13 INFO  ==============================================================
13:34:13 INFO  Id:    mapper_awaits_before_reconnect
13:34:13 INFO  Dir:   tedge_mapper_c8y/
13:34:13 INFO  Title: Validate mapper doesn't reconnect too often
13:34:13 INFO  ==============================================================
13:34:13 INFO  Started mosquitto_sub with process id 9190
13:34:13 INFO  Waiting for 0.1 seconds
13:34:14 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
13:34:14 INFO  Waiting for 5.0 seconds
13:34:20 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
13:34:21 INFO  Executed sudo&lt;tedge_mapper_journal&gt;, exit status 0
13:34:21 WARN  Grep on tedge_mapper_journal.out contains "ERROR" ... failed [run.py:67]
13:34:21 INFO  Line count on file tedge_mapper_journal.out ... passed
13:34:21 WARN  Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0 ... failed [run.py:78]
13:34:21 WARN  Aborted test due to FAILED outcome
13:34:21 INFO  
13:34:21 INFO  Test duration: 7.63 secs
13:34:21 INFO  Test final outcome:  FAILED
13:34:21 INFO  Test outcome reason: Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0
13:34:21 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_smaller_interval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringSmallInterval" name="monitoring_smaller_interval">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>13:21:08 INFO  ==============================================================
13:21:08 INFO  Id:    monitoring_smaller_interval
13:21:08 INFO  Dir:   monitoring/
13:21:08 INFO  Title: Validate the tedge dm agent messages that are produced with smaller sampling interval than the batching window. Here the simulated collectd messages are used
13:21:08 INFO  ==============================================================
13:21:08 INFO  Executed sudo&lt;collectd&gt;, exit status 0
13:21:09 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
13:21:09 INFO  Started sudo&lt;tedge_sub&gt; with process id 5339
13:21:10 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:10 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:11 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:11 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:12 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:12 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:13 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:14 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:14 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:15 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:15 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:16 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:16 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:17 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:18 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:18 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:19 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:19 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:20 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:21 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:22 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
13:21:22 INFO  0

13:21:22 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 104, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 114, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 134, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
13:21:22 WARN  TypeError: 'int' object is not subscriptable ... blocked 
13:21:22 INFO  
13:21:22 INFO  cleanup:
13:21:22 INFO  monitoring_cleanup
13:21:23 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
13:21:23 INFO  
13:21:23 INFO  Test duration: 15.12 secs
13:21:23 INFO  Test final outcome:  BLOCKED
13:21:23 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
13:21:23 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_collectd" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MonitoringWithCollectd" name="monitoring_with_collectd">
		<skipped message="SKIPPED: Test execution disabled due to #792 (instable)" />
		<system-out>13:21:23 INFO  ==============================================================
13:21:23 INFO  Id:    monitoring_with_collectd
13:21:23 INFO  Dir:   monitoring/
13:21:23 INFO  Title: Validate tedge dm agent messages that are produced by batching the collectd messages
13:21:23 INFO  ==============================================================
13:21:23 INFO  Test execution disabled due to #792 (instable) ... skipped
13:21:23 INFO  
13:21:23 INFO  Test duration: 0.02 secs
13:21:23 INFO  Test final outcome:  SKIPPED
13:21:23 INFO  Test outcome reason: Test execution disabled due to #792 (instable)
13:21:23 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_simulated_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringWithSimulatedMessages" name="monitoring_with_simulated_messages">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>13:21:23 INFO  ==============================================================
13:21:23 INFO  Id:    monitoring_with_simulated_messages
13:21:23 INFO  Dir:   monitoring/
13:21:23 INFO  Title: Validate the tedge dm agent messages that are produced from the simulated collectd messages
13:21:23 INFO  ==============================================================
13:21:24 INFO  Executed sudo&lt;collectd&gt;, exit status 0
13:21:24 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
13:21:24 INFO  Started sudo&lt;tedge_sub&gt; with process id 5399
13:21:25 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:25 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:27 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
13:21:27 INFO  0

13:21:27 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 98, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 108, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 127, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
13:21:27 WARN  TypeError: 'int' object is not subscriptable ... blocked 
13:21:27 INFO  
13:21:27 INFO  cleanup:
13:21:27 INFO  monitoring_cleanup
13:21:27 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
13:21:27 INFO  
13:21:27 INFO  Test duration: 4.48 secs
13:21:27 INFO  Test final outcome:  BLOCKED
13:21:27 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
13:21:27 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_change_connection_fails" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortChangeConnectionFails" name="mqtt_port_change_connection_fails" />
</testsuite><testsuite name="mqtt_port_change_connection_works" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MqttPortChangeConnectionWorks" name="mqtt_port_change_connection_works">
		<skipped message="SKIPPED: failing intermitently" />
		<system-out>13:20:12 INFO  ==============================================================
13:20:12 INFO  Id:    mqtt_port_change_connection_works
13:20:12 INFO  Dir:   misc_features/
13:20:12 INFO  Title: mqtt port change connection works for all the thin-edge services that use the mosquitto broker
13:20:12 INFO  ==============================================================
13:20:12 INFO  failing intermitently ... skipped
13:20:12 INFO  
13:20:12 INFO  Test duration: 0.02 secs
13:20:12 INFO  Test final outcome:  SKIPPED
13:20:12 INFO  Test outcome reason: failing intermitently
13:20:12 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_set" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortSet" name="mqtt_port_set" />
</testsuite><testsuite name="mqtt_port_unset" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortUnSet" name="mqtt_port_unset" />
</testsuite><testsuite name="publish_flux" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishFlux" name="publish_flux" />
</testsuite><testsuite name="publish_sawmill" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishSawmill" name="publish_sawmill" />
</testsuite><testsuite name="runtime_start_two_mappers" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RuntimeMultiMappers" name="runtime_start_two_mappers" />
</testsuite><testsuite name="sm_tedge_mapper_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeSMMapperInitSession" name="sm_tedge_mapper_init_session" />
</testsuite><testsuite name="smoketest_json" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmokeTestJson" name="smoketest_json" />
</testsuite><testsuite name="smoketest_json_400_samples_10ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples10ms" name="smoketest_json_400_samples_10ms" />
</testsuite><testsuite name="smoketest_json_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples20ms" name="smoketest_json_400_samples_20ms" />
</testsuite><testsuite name="smoketest_smartrest" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest" name="smoketest_smartrest" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_1Xms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples1Xms" name="smoketest_smartrest_400_samples_1Xms" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples20ms" name="smoketest_smartrest_400_samples_20ms" />
</testsuite><testsuite name="tedge_agent_user_sudo_access" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="TedgeAgentUserSudoAccess" name="tedge_agent_user_sudo_access">
		<skipped message="SKIPPED: #955 Observation: Exit code of the apt plugin when called with no arguments" />
		<system-out>13:31:36 INFO  ==============================================================
13:31:36 INFO  Id:    tedge_agent_user_sudo_access
13:31:36 INFO  Dir:   tedge/
13:31:36 INFO  Title: Validate tedge-agent user has a limited sudo right
13:31:36 INFO  ==============================================================
13:31:36 INFO  #955 Observation: Exit code of the apt plugin when called with no arguments ... skipped
13:31:36 INFO  
13:31:36 INFO  Test duration: 0.02 secs
13:31:36 INFO  Test final outcome:  SKIPPED
13:31:36 INFO  Test outcome reason: #955 Observation: Exit code of the apt plugin when called with no arguments
13:31:36 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_connect_test_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestNegative" name="tedge_connect_test_negative" />
</testsuite><testsuite name="tedge_connect_test_positive" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestPositive" name="tedge_connect_test_positive" />
</testsuite><testsuite name="tedge_connect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestSMServices" name="tedge_connect_test_sm_services" />
</testsuite><testsuite name="tedge_disconnect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeDisConnectTestSMServices" name="tedge_disconnect_test_sm_services" />
</testsuite><testsuite name="tedge_mapper_az_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzError" name="tedge_mapper_az_negative" />
</testsuite><testsuite name="tedge_mapper_az_with_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithTimestamp" name="tedge_mapper_az_with_timestamp" />
</testsuite><testsuite name="tedge_mapper_az_without_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithoutTimestamp" name="tedge_mapper_az_without_timestamp" />
</testsuite><testsuite name="tedge_mapper_c8y_alarms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yAlarm" name="tedge_mapper_c8y_alarms" />
</testsuite><testsuite name="tedge_mapper_c8y_child_device_measurement" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yChildDevice" name="tedge_mapper_c8y_child_device_measurement" />
</testsuite><testsuite name="tedge_mapper_c8y_packet_threshold_size" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="TedgeMapperC8yThresholdPacketSize" name="tedge_mapper_c8y_packet_threshold_size">
		<failure message="FAILED: Grep on tedge_sub.out contains &quot;The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184.&quot;" type="FAILED" />
		<system-out>13:35:58 INFO  ==============================================================
13:35:58 INFO  Id:    tedge_mapper_c8y_packet_threshold_size
13:35:58 INFO  Dir:   tedge_mapper_c8y/
13:35:58 INFO  Title: Validate the Cumulocity Mapper accepted threshold packet size when it receives a message on tedge/measurements
13:35:58 INFO  ==============================================================
13:35:59 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
13:35:59 INFO  Started sudo&lt;tedge_sub&gt; with process id 9791
13:35:59 INFO  Executed sudo&lt;tedge_msg&gt;, exit status 0
13:36:00 INFO  Executed sudo&lt;tedge_pub&gt;, exit status 0
13:36:00 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
13:36:00 WARN  Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184." ... failed [run.py:88]
13:36:00 INFO  
13:36:00 INFO  cleanup:
13:36:00 INFO  mapper_cleanup
13:36:01 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
13:36:01 INFO  
13:36:01 INFO  Test duration: 3.00 secs
13:36:01 INFO  Test final outcome:  FAILED
13:36:01 INFO  Test outcome reason: Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184."
13:36:01 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_software_update" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="tedge_software_update" />
</testsuite><testsuite name="valid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateValidDeviceId" name="valid_device_id" />
</testsuite></testsuites>